builtin/logical/pki/backend_test.go

```diff
@@ -2522,7 +2522,7 @@ func TestBackend_SignSelfIssued(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	signingBundle, err := fetchCAInfo(context.Background(), b, &logical.Request{Storage: storage})
+	signingBundle, err := fetchCAInfo(context.Background(), b, &logical.Request{Storage: storage}, "default")
```

Comment:
Could we define a constant for "default"? Maybe a typed string of some sort. 
Basically please protect me from the inevitable typo I will write ;)

Reply:
This is actually from #14805 -- if you see what I've done in some of the other commits (`sign-verbatim`, `sign`, `sign-intermediate`, &cp -- you'll see that we're mostly updating it to:

> ```go
> signingBundle, caErr := fetchCAInfo(ctx, b, req, issuerName)
> ```

and `"default"` is the default value of [`addNonCACommonFields`](https://github.com/hashicorp/vault/pull/14847/files#diff-22663f59eb7343dfccf79ce2731482c07b5d867d2a8942a7b81270d34a9c6f91R135)'s `ref` field.

Reply:
Okay we can adopt a wait and see approach.

Reply:
Sorry, I've changed the branch of that PR now so its clearer. :D 

 - There's the weird `backend_test.go` one, not sure what I'd do with that one.
 - `crl_util.go` will be handled by the later PR, that needs to be based on revoked cert's actual issuer.
 - `path_fetch.go` is the exception I mentioned above,
 - And both `path_issue_sign` and `path_root.go` were updated in this PR.